### DISCUSSION TITLE
the message does not arrive to whatsapp

### DISCUSSION THREAD

**Discussion by HadyCloud** [Zain-ul-din/whatsapp-ai-bot#11] [Q&A]
after i send !chatgpt xxxxon whatsapp, i can see on the command prompt that a response arrived with success , yet in whatsapp I cant see the reply ... help ?

**Comment by Zain-ul-din** [comment]
@HadyCloud Please share more detail like `screenshots` or `response text` so we can figure out the problem.

> after i send !chatgpt xxxxon whatsapp, i can see on the command prompt that a response arrived with success , yet in whatsapp I cant see the reply ... help ?

There could be some potentials reasons why the response message is not arriving in whats app

- you may forget to add `chat-gpt api key`.
- maybe the message fail due to `connection` problem.
- you might scan the code from a different device than you are testing on.
